build_tag_push.py

```diff
@@ -23,54 +23,33 @@ def parseAddFile(filename):
-# Creation. In the DockerFile, you generally do: FROM = tells Docker which image you base your image on, 
+# They will have their own Dockerfile in their GitHub repo which will then be placed in our MVP Studio dir
+# Recall: Dockerfile -> docker build -> Docker Image, it is basically an automation of Docker Image Creation. 
+# In the DockerFile, you generally do: FROM = tells Docker which image you base your image on, 
 # RUN = tells Docker which additional commands to execute, CMD = tells Docker to execute the command when the image loads.
 # You then need to build the Dockerfile to create the image and run image (this is the ID not the name) to create the container
 
 def build_DockerImageandTag(filename):
-	""" Helpful sources:
+	""" Sources:
 	https://docs.docker.com/compose/gettingstarted/#step-3-define-services-in-a-compose-file
 	https://docs.docker.com/engine/reference/commandline/commit/
 	https://runnable.com/docker/python/dockerize-your-python-application
 	https://www.youtube.com/watch?v=LQjaJINkQXY
 	"""
-	os.environ['IMAGE_NAME'] = parseAddFile(filename)[0] # This is creating the environment variable $IMAGE_NAME based
-	# off of the add.yml file that is in their GitHub repo and since I am returning their add.yml contents as a list
-	# in my parseAddFile function, I am indexing that list to get the name out.
-	os.system("echo $IMAGE_NAME") # This is just a check to make sure I am getting the correct result. Success.
-	os.system("docker build -t $IMAGE_NAME .") 
+	image_name = parseAddFile(filename)[0] # Name from app.yml file
+	subprocess.check_call(["docker", "build", image_name, "."]) # Terminal command: docker build -t $IMAGE_NAME .
 	# This . means that you are in the current directory holding the Dockerfile that you need.
-	# You can give an exact location instead of the . so for my setup it would be /Users/alyssakelley/Desktop/MVP Studio/DockerFiles
-	# You can have the -t flag to tag the image. So it would be: docker build -t imagename:tag . 
-	# Note: Then if you want to run that Docker image that was just created, you can do the following command in the terminal:
-	# 	docker run <image ID> 
-	# Note: The Image ID you can access with the command 'docker images'
-	image_id = os.system("docker images -q $IMAGE_NAME")
-	print image_id
-	#os.environ['IMAGE_ID'] = image_id  # This -q flag retrieves the last image ID (Doesn't work... TypeError)
-	#os.system("docker run $IMAGE_ID")
-	#os.system("docker run docker images -q $IMAGE_NAME") -> Nope.
-	#os.system("docker images -q $IMAGE_NAME")
-	#os.system("IMAGE_ID=$(docker images -q $IMAGE_NAME)")
-	os.system("echo done")
-	#os.system('docker run $IMAGE_ID')
+	image_id = subprocess.check_call(["docker", "images", "-q", image_name]) # Error here: This ID is still being saved as an int... I will be working on fixing this.
+	subprocess.check_call(["docker", "run", image_id])
```

Comment:
This is cool that you figured out how to grab the image id and run it!
I didn't know about the `-q` flag.
Unfortunately, we need to do a docker push instead.
`docker push <image-name>:latest` would probably work for now.
Later we will need to do a `docker push` for each tag that we create (the version, the git hash, etc...)